NitroxServer/GameLogic/TimeKeeper.cs

```diff
@@ -1,63 +1,59 @@
 ﻿using System;
-using NitroxModel.Core;
 using NitroxModel.Packets;
 
 namespace NitroxServer.GameLogic
 {
     public class TimeKeeper
     {
-        // Values taken directly from hardcoded subnautica values
-        private static readonly DateTime SUBNAUTICA_FUTURE_START_DATE = new DateTime(2287, 5, 7, 9, 36, 0);
+        public readonly double SUBNAUTICA_BEGIN_TIME_OFFSET;
+        public readonly DateTime SUBNAUTICA_DATE_ORIGIN;
 
-        private static readonly float SUBNAUTICA_BEGIN_TIME_OFFSET = 1200f /
-                                                                     86400f *
-                                                                     (3600f * SUBNAUTICA_FUTURE_START_DATE.Hour +
-                                                                      60f * SUBNAUTICA_FUTURE_START_DATE.Minute +
-                                                                      SUBNAUTICA_FUTURE_START_DATE.Second);
+        public DateTime ServerStartTime { get; set; }
 
-        // Discrepancy value for player based time modifications
-        private float correctionValue;
+        private readonly PlayerManager playerManager;
+        private double correctionValue;
 
-        public DateTime ServerStartTime { get; set; } = DateTime.UtcNow;
+        public TimeKeeper(PlayerManager playerManager)
```

Comment:
Does this change now work, where does playermanager get set?

Reply:
Dependency injection